apps/docs/tsconfig.json

```diff
@@ -1,8 +1,10 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "outDir": "./dist"
+    "module": "ES2022",
```

Comment:
Just confirming but this is to enable top-level await?

Reply:
Correct :)